test/Adapter/Driver/IbmDb2/StatementIntegrationTest.php

```diff
@@ -86,6 +87,23 @@ public function testPrepare()
     }
 
     /**
+     * @covers Zend\Db\Adapter\Driver\IbmDb2\Statement::prepare
+     * @expectedException RuntimeException
```

Comment:
I would move it to function use, because right now it's not clear, which line should thrown that exception.

Reply:
@webimpress I can't - it doesn't work on the AS/400 when I do that. I have no idea why.

Reply:
Hm... so maybe exception is thrown not where you expecting it?

Reply:
It was a PHP error. I'll have another look when I'm next connected to that computer.